libr/io/io_fd.c

```diff
@@ -126,3 +126,39 @@ R_API int r_io_fd_get_current(RIO *io) {
 	}
 	return -1;
 }
+
+R_API int r_io_fd_get_next(RIO *io, int fd) {
+	r_return_val_if_fail (io, -1);
+	int ret = fd;
+	if (!r_id_storage_get_next (io->files, (ut32 *)&ret)) {
+		return -1;
+	}
+	return ret;
+}
+
+R_API int r_io_fd_get_prev(RIO *io, int fd) {
+	r_return_val_if_fail (io, -1);
+	int ret = fd;
+	if (!r_id_storage_get_prev (io->files, (ut32 *)&ret)) {
+		return -1;
+	}
+	return ret;
+}
+
+R_API int r_io_fd_get_highest(RIO *io) {
+	r_return_val_if_fail (io, -1);
+	int fd = 0;
+	if (!r_id_storage_get_highest (io->files, (ut32 *)&fd) && fd == 0) {
+		return -1;
+	}
+	return fd;
+}
+
+R_API int r_io_fd_get_lowest(RIO *io) {
+	r_return_val_if_fail (io, -1);
+	int fd = 0;
+	if (!r_id_storage_get_lowest (io->files, (ut32 *)&fd) && fd == 0) {
```

Comment:
```suggestion
	if (!r_id_storage_get_lowest (io->files, (ut32 *)&fd)) {
```

Reply:
I imagine this is about to check if io->files is empty

Reply:
@trufae if this is the intention here, we can add r_id_storage_is_empty

Reply:
done

Reply:
I still see `fd == 0` @trufae